app/src/theme/index.tsx

```diff
@@ -387,17 +387,26 @@ function themeAggregator(darkMode: boolean) {
     ...colors(darkMode),
   }
 }
+
 export function ThemeProvider({ children }: { children: React.ReactNode }) {
   //this is temporary until we determine to use dark theme
   const darkMode = false
 
   return <StyledThemeProvider theme={themeAggregator(darkMode)}>{children}</StyledThemeProvider>
 }
 
-const TextWrapper = styled.div<{ color: keyof Colors }>`
-  color: ${props => props.theme.text3};
+const TextWrapper = styled.div<TextProps>`
+  color: ${({ color, theme }) => (theme as any)[color]};
   font-family: Roboto;
-  letter-spacing: 0.2px;
+  font-size: ${props => props.fontSize};
+  font-weight: ${props => props.fontWeight};
+  letter-spacing: ${props => (props.letterSpacing ? props.letterSpacing : '0.2px')};
+  line-height: ${props => props.lineHeight};
+  ${props => props.margin && `margin: ${props.margin}`};
+  ${props => props.marginBottom && `margin-bottom: ${props.marginBottom}`};
+  ${props => props.marginLeft && `margin-left: ${props.marginLeft}`};
+  ${props => props.marginTop && `margin-top: ${props.marginTop}`};
+  ${props => props.display && `display: ${props.display}`};
```

Comment:
Hey men so this is not a way to go about it....The last parameter for each Type {...props} is supposed to do that what you wrote here. It basically passes all the properties down to the text type. If it doesn't work like that then we need to adjust it to work...